examples/full.py

```diff
@@ -3,178 +3,230 @@
 import re
 
 from flask import Flask
-from slackify import (ACK, OK, Slackify, async_task, block_reply,
-                      request, respond, text_block, Slack)
-
+from slackify import (
+	ACK, OK, Slackify, async_task, block_reply,
+	request, respond, text_block, reply_text, Slack)
 
 app = Flask(__name__)
-slackify = Slackify(app=app)
+slackify = Slackify(app = app)
 cli = Slack(os.getenv('BOT_TOKEN'))
 
 
 @slackify.command
 def hello():
-    YES = 'yes'
-    NO = 'no'
-    yes_no_buttons_block = {
-        "type": "actions",
-        "elements": [
-            {
-                "type": "button",
-                "text": {
-                    "type": "plain_text",
-                    "emoji": True,
-                    "text": "Yes"
-                },
-                "style": "primary",
-                "value": "i_like_bots",
-                "action_id": YES
-            },
-            {
-                "type": "button",
-                "text": {
-                    "type": "plain_text",
-                    "emoji": True,
-                    "text": "No"
-                },
-                "style": "danger",
-                "value": "i_dont_like_bots",
-                "action_id": NO
-            }
-        ]
-    }
-    blocks = [
-        text_block('Do you like Bots?'),
-        yes_no_buttons_block
-    ]
-    return block_reply(blocks)
+	YES = 'yes'
```

Comment:
Shouldn't this be 4 spaces?

Reply:
All the spaces issues should be sorted out on the next push 😅